CMakeLists.txt

```diff
@@ -40,6 +40,12 @@ if(NOT CMAKE_BUILD_TYPE)
 	FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
+if (NOT SHARE_INSTALL_PREFIX)
+	set(SHARE_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX}/share)
```

Comment:
please use `GnuInstallDirs` here, i.e. `CMAKE_INSTALL_DATAROOTDIR`

Reply:
Oh yes, that's what I was looking for but couldn't find. Thanks!